src/libs/network/manageNodeCall.ts

```
@@ -91,15 +92,16 @@ export async function manageNodeCall(content: NodeCall): Promise<RPCResponse> {
         case "getPeerlist":
             response.response = await getPeerlist()
             break
-        case "getPeerlistHash":
-            var peerlist = await getPeerlist()
+        case "getPeerlistHash": {
+            let peerlist = await getPeerlist()
```

ESLint

[error] 'peerlist' is never reassigned. Use 'const' instead.

(prefer-const)